db/dynamodb.py

```diff
@@ -16,10 +18,41 @@ class DynamoDB:
     """
 
     def __init__(self):
-        """Initialize facade using DynamoDB settings (for now)."""
-        # TODO change this to production and not localhost
-        self.ddb = boto3.resource("dynamodb", region_name="",
-                                  endpoint_url="http://localhost:8000")
+        """Initialize facade using DynamoDB settings.
+
+        To avoid local tests failure when the dyanmodb server is used,
+        a testing environment variable is set.
+        When testing environmental variable is true,
+        the local dynamodb is run.
+        When testing environmental variable is true,
+        the server dynamodb is run.
+
+        boto3.resource() takes in a service_name, region_name, and endpoint_url
+        (only for local dynamodb).
+        service_name: The name of a service, "dynamodb" in this case.
+        region_name:  The name of the region associated with the client.
+        A list of different regions can be obtained online.
+        endpoint_url: The complete URL to use for the constructed client.
+
+        Boto3 server require environmental variables for credentials:
+        AWS_ACCESS_KEY_ID: The access key for your AWS account.
+        AWS_SECRET_ACCESS_KEY: The secret key for the AWS account
+        AWS_SESSION_TOKEN: The session key for your AWS account.
+        This is only needed when you are using temporary credentials.
+        """
+        testing = bool(os.environ.get("TESTING", True))
+        region_name = os.environ.get("REGION", '')
+        if region_name == '':
```

Comment:
This can be squashed into the call to `os.environ.get()` - the second parameter is the default value to use.